### ISSUE TITLE
 Unable to load library: webviewchromium

### ISSUE THREAD

**Issue by abhinavkumar17** [chat-sdk/chat-sdk-android#733]
1. **Is the bug present in the demo Chat SDK project?**
Yes
2. **What modifications have you made to the Chat SDK?**
None
3. **Android Version:**
8.0
4. **Steps taken to reproduce the problem:**
Install app on android device 8, setup login and add contact. 
Click on contact to start conversation and observer crash.
5. **Expected result:**

6. **Actual result:**

7. **Comments:**
022-05-23 11:48:48.573 15224-15224/com.comminication.chat.mesanger E/cr_LibraryLoader: Unable to load library: webviewchromium
2022-05-23 11:48:48.588 15224-15224/com.comminication.chat.mesanger E/WebViewFactory: error instantiating provider
    java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at android.webkit.WebViewFactory.getProvider(WebViewFactory.java:208)
        at android.webkit.WebView.getFactory(WebView.java:2530)
        at android.webkit.WebView.findAddress(WebView.java:1802)
        at android.text.util.Linkify.gatherMapLinks(Linkify.java:562)
        at android.text.util.Linkify.addLinks(Linkify.java:260)
        at android.text.util.Linkify.addLinks(Linkify.java:226)
        at android.widget.TextView.setText(TextView.java:5330)
        at android.widget.TextView.setText(TextView.java:5250)
        at android.widget.TextView.<init>(TextView.java:1626)
        at android.widget.TextView.<init>(TextView.java:820)

**Comment by bensmiley** [comment]
@abhinavkumar17 this error is coming from a project called: 

```
com.comminication.chat.mesanger
```

Which means that it's not the demo project... Maybe you have integrated this with your own project? Can you let me know the Chat SDK version and how it's been integrated so I can try to help.

**Comment by abhinavkumar17** [comment]
Hi @bensmiley 

Here is the steps: 
I created new project and done all setups in accordance to this document:

https://chat-sdk.gitbook.io/android/setup/add-chat-sdk-to-your-project

Now to to test this I have used 2 devices running on android 12 and I was able to successfully able to chat.
We have some android device with OS version8. When we tried to use this device as the 3rd test device we get this error.
The crash is observed at the point where I select a user to chat with which already has some messages. It works fine where there are no messages. Crashes when message is sent/received.
[New Text Document.txt](https://github.com/chat-sdk/chat-sdk-android/files/8766073/New.Text.Document.txt)
